templates/index.html

```diff
@@ -97,8 +35,8 @@ <h2>Cloud-style VMs on-demand</h2>
   <div class="row u-equal-height u-vertically-center">
```

Comment:
I think we could also make this section just a `p-strip` as it doesn't look right how it currently is